ENIGMAsystem/SHELL/Widget_Systems/General/tinyfiledialogs/dialogs.cpp

```diff
@@ -260,6 +273,8 @@ int show_message(const string &str) {
   msg = tfd_add_escaping(msg);
   caption = tfd_add_escaping(caption);
 
+  window_activate();
```

Comment:
As the number of repeated lines continues to stack up, you might consider extracting an initialize helper of some sort.